src/components/iconsBackground.jsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import "../styles/Background.scss";
+
+const IconsBackground = () => {
+  return (
+    <React.Fragment>
+      <div className="container__icons-1">
```

Comment:
Esto tiene un patron de iconos que se repite? si es asi, por que no usas un for?

Reply:
los ordene random, por eso los deje asi, son dos filas con los iconos ordenados de distinta forma para que no me queden todas las columnas repetidas durante la animacion